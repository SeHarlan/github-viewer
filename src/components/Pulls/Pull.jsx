import React from 'react';
import PropTypes from 'prop-types';

const Pull = ({ pullBody, pullNumber }) => (
  <em>{pullNumber}: {pullBody}</em>
);

Pull.propTypes = {
  pullBody: PropTypes.string.isRequired,
  pullNumber: PropTypes.number.isRequired
};

export default Pull;
