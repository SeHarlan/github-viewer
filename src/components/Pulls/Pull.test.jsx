import React from 'react';
import { shallow } from 'enzyme';
import Pull from './Pull.jsx';

describe('Pull component', () => {
  it('renders Pull', () => {
    const wrapper = shallow(<Pull pullBody="pullbody" pullNumber={2}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
