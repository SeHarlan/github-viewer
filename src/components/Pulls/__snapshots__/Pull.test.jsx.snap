// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pull component renders Pull 1`] = `
<em>
  2
  : 
  pullbody
</em>
`;
